src/service/extensions-impl.js

```diff
@@ -107,6 +109,14 @@ let ExtensionDef;
  */
 let ExtensionHolderDef;
 
+/**
+ * @const {{experiment: string, control: string, branch: string}}
+ */
+const FIE_CSS_CLEANUP_EXP = {
```

Comment:
double-click and adsense impl should import the setup from here, instead of duplicating the constants.